### PR TITLE
Sidenav toggle button

### PR DESCRIPTION
**Pull request by kenfdev**
Signed-off-by: Ken Fukuyama <kenfdev@gmail.com>

## Description

The current gateway UI's `sidenav` hides when the screen's width is below 960px. I've added a toggle menu button when the screen's width is below 960px. Now you can toggle the `sidenav` even though the screen is small. The function is as below.

toggle_sidenav

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/alexellis/faas/blob/master/CONTRIBUTING.md))
The issue is mention at #180 .

## How Has This Been Tested?

I've tested this by deploying it on a Rancher backed OpenFaaS environment and opening up the gateway UI. Shrinking and stretching it to see if the toggle menu works correctly. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: gateway/assets/img/icons/menu.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="24" height="24" viewBox="0 0 24 24"><path d="M3,6H21V8H3V6M3,11H21V13H3V11M3,16H21V18H3V16Z" /></svg>
```
Comment (alexellis) [inline gateway/assets/img/icons/menu.svg]: Where did you get `menu.svg` from? Do we have usage rights?

**Comment by alexellis** [comment]
I'm wondering if we should dock/pin when you click the menu icon and allow it to be undocked/unpinned when clicked again?

File: gateway/assets/img/icons/menu.svg (hunk 1 above)
Comment (kenfdev) [reply gateway/assets/img/icons/menu.svg]: Sorry @alexellis , should have pasted the link here, too. I got the icons from [here](https://github.com/Templarian/MaterialDesign). I though it was the official Material Design Icon but it seems it's a community driven one. It's under Apache License 2.0 so it's usable, but I'll replace it with the [official one](https://github.com/google/material-design-icons). (Which is also under Apache License 2.0)

**Comment by kenfdev** [comment]
@alexellis I understand. I'll give it a try today.

**Comment by alexellis** [comment]
Thanks @kenfdev .. looking forward to seeing it merged and in a coming release.

**Comment by kenfdev** [comment]
@alexellis Sorry it took time, how about something like this? I added a "Navigate back" button.

faas_sidenav

**Comment by alexellis** [comment]
I think we may need to tweak again but I'd like to merge so we can get some feedback. Thank you for helping 👍

**Pull request merged**